src/features/Ens/EnsTable.tsx

```diff
@@ -20,18 +20,23 @@ const EnsTableContainer = styled.div`
   overflow: auto;
 `;
 
-export const EnsTable = ({ records, isFetched }: MyDomainsData) => (
-  <EnsTableContainer>
-    {records.length === 0 &&
-      (isFetched ? (
-        <NoDomains />
-      ) : (
-        <SpinnerContainer>
-          <Spinner size={'x3'} />
-        </SpinnerContainer>
-      ))}
-    {records.length !== 0 && <MyDomains domainOwnershipRecords={records} />}
-  </EnsTableContainer>
-);
+export const EnsTable = () => {
+  const { ensOwnershipRecords, isEnsFetched } = useContext(StoreContext);
```

Comment:
I think we should fetch the data in `EnsDashboard` and keep this a dumb component.